ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -65,10 +65,10 @@ jobs:
     requires: [fedora-latest/build]
     priority: 50
     job:
-      class: RunPytest
+      class: RunADTests
```

Comment:
The currently gating configuration for test_ipahealthcheck uses RunPytest. Do we need to update those definitions as well?

Reply:
test_ipahealthcheck::TestIpaHealthCheckWithADtrust has been updated in all required yaml files.